### PR TITLE
Updated libassuan to v 2.5.1, plus added support to put libraries in lib64 for x86_64

### PR DESCRIPTION
**Pull request by duckunix**
x86_64 hosts

Supports #1585 

## Description
Updated libassuan to v2.5.1.  Also, added support to put library files in .../lib64 for x86_64 hosts
## Additional information

Built & tested on:
- [x] x86_64
- [x]  armv7l
- [x] i686 (No hardware to test)
- [ ] aarch64 (No hardware to test)

 Also, removed pre-built stanzas

### CODE REVIEW

File: packages/libassuan.rb [hunk 1]
```diff
@@ -3,25 +3,17 @@
...
+  source_sha256 '47f96c37b4f2aac289f0bc1bacfa8bd8b4b209a488d3d15e2229cb6cc9b26449'
 
   def self.build
-    system './configure --prefix=/usr/local'
+    case ARCH
+      when 'x86_64'
+        system './configure --prefix=/usr/local --libdir=/usr/local/lib64'
+      else
+        system './configure --prefix=/usr/local'
+    end
```
Comment (uberhacker) [inline packages/libassuan.rb]: You can replace this entire conditional logic using crew constants with a single line:
```
system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
```

**Comment by uberhacker** [comment]
@duckunix: Thank you for the update!  Please see comment in the `Files changed` section.

File: packages/libassuan.rb (hunk 1 above)
Comment (duckunix) [reply packages/libassuan.rb]: Still learning ruby and chromebrew way.  That does make for cleaner code...I will update...

**Comment by duckunix** [comment]
Updated per comment above.

**Comment by uberhacker** [comment]
While testing on i686, I noticed there was a missing dependency:
```
checking for GPG Error - version >= 1.17... no
configure: error: libgpg-error was not found
libassuan failed to install: `./configure --prefix=/usr/local --libdir=/usr/local/lib` exited with 1
```
It appears we need `depends_on 'libgpgerror'` added.

**Comment by duckunix** [comment]
Added depends_on 'libgpgerror', still builds for me. :)

**Comment by uberhacker** [comment]
I didn't test on aarch64 but most of the time if it works on armv7l, it works on aarch64.

**Pull request merged**